src/framework/mlt_frame.c

```diff
@@ -410,10 +410,35 @@ const char * mlt_image_format_name( mlt_image_format format )
 		case mlt_image_opengl:  return "opengl";
 		case mlt_image_glsl:    return "glsl";
 		case mlt_image_glsl_texture: return "glsl_texture";
+		case mlt_image_yuv422p16: return "yuv422p16";
+		case mlt_image_last:    return "last";
```

Comment:
I think that last should return "invalid" to be consistent with previous behavior. But I'm not sure.

Alternately, maybe we should remove "invalid" and always return "last" as a default.

Either way, I don't think there should be two different error returns. It should either be "invalid" or "last".